tools/ci/update-vulnerable-dependencies.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+command -v osv-scanner >/dev/null 2>&1 || { echo >&2 "osv-scanner not installed!"; exit 1; }
+command -v jq >/dev/null 2>&1 || { echo >&2 "jq not installed!"; exit 1; }
```

Comment:
[yq](https://github.com/kumahq/kuma/blob/6f85a6390a3b928d20b50faeeeb57f255deb3187/mk/dependencies/go-deps.sh#L18) is in dev tools dir, probably we could pull jq there as well, but not to clog this up we could just create an issue and address that in the future.

Reply:
See above